src/pages/profiles/[userId].tsx

```diff
@@ -258,15 +264,36 @@ function Picture({ userId, profile, updateProfile, SaveButton }: {
     [userId, profile]
   );
 
+  const updateImageParams = (x: number, y: number, zoom: number) => {
+    updateProfile('照片参数', { x, y, zoom }); 
+  };
+
   return <>
     <Heading size="md">生活照</Heading>
     <FormControl>
-      {profile.照片链接 && <Image
-        src={profile.照片链接}
-        alt="照片"
-        maxW='300px'
-        my={componentSpacing}
-      />}
+      {!profile.照片参数 && <FullWidthImageSquare profile={profile}/>}
+      {profile.照片链接 && profile.照片参数 &&
+        <Box
+          position="relative"
+          width="300px"
+          height="300px"
+          overflow="hidden"
+        >
+          <Image
+            width="300px"
+            height="300px"
+            position="absolute"
+            transform={`
+              translate(${(profile.照片参数.x)}px, 
+                        ${(profile.照片参数.y)}px)
+              scale(${profile.照片参数.zoom})
+            `}
+            transformOrigin="center center"
+            objectFit="contain"
+            src={profile.照片链接}
+            alt="照片"
+          />
+      </Box>}
```

Comment:
modify `FullWidthImageSquare` to take in `ImageParams` as an optional property and crop the image inside `FullWidthImageSquare`. When `ImageParams` is absent, use `x:0, x:0, zoom:1` as the default property.

Reply:
Also, use a Box to bound FullWidthImageSquare to 300px